api/services/Search.js

```diff
@@ -37,6 +37,10 @@ module.exports = {
         qb.orderBy(opts.sort || '', opts.order || 'asc')
       }
 
+      if (opts.is_public) {
+        qb.whereIn('is_public', opts.is_public)
+      }
+
```

Comment:
looks like you still need to add the boundingBox part right?

Reply:
added ✔️